integrations/dd-sdk-android-okhttp/src/test/kotlin/com/datadog/android/okhttp/trace/TracingInterceptorTest.kt

```diff
@@ -1567,6 +1571,27 @@ internal open class TracingInterceptorTest {
         assertThat(called).isEqualTo(1)
     }
 
+    @Test
+    fun `M add trace sample rate to tracing feature context W constructor called`(
+        @FloatForgery(min = 0f, max = 100f) sampleRate: Float
```

Comment:
minor: usually in tests there fake values are called `fakeXXX`, e.g. here it will be `fakeSampleRate`.

But no need to update PR, it is good like that.